ports/raspberrypi/boards/boardsource_blok/mpconfigboard.mk

```diff
@@ -0,0 +1,14 @@
+USB_VID = 0x4273
```

Comment:
What vendor is this?

Reply:
yeah so this is just the one we at boardsource used for our keyboards in [qmk](https://github.com/qmk/qmk_firmware/tree/master/keyboards/boardsource/) I swore it ascii table for bs(boardsource) or something but I cant figure out now how we came up with this number. What do we need to put there? We are happy to change it.

Reply:
@tannewt I believe this to be resolved in the latest rebase.
https://github.com/raspberrypi/usb-pid